docs/source/developing/writing-docs.adoc

```diff
@@ -110,4 +183,7 @@ $ cd ..
 $ rake build
 ----
 
-If the build completes successfully, the site is available under _preview/openshift-origin/latest/welcome/index.html_.
+If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+OpenShift runs a nightly build in which they update the documentation. After you update the documentation,
```

Comment:
Ok like I said, I didn't actually add this information so it was wrong before and I'll be happy to update it so that it's accurate. I was wondering myself how the missing piece of the process comes into play, so it's good that we're fixing this now.